10-habits/progress/api/service_grpc.pb.go

```diff
@@ -59,17 +65,40 @@ func (c *habitsClient) ListHabits(ctx context.Context, in *ListHabitsRequest, op
 	return out, nil
 }
 
+func (c *habitsClient) TickHabit(ctx context.Context, in *TickHabitRequest, opts ...grpc.CallOption) (*TickHabitResponse, error) {
+	out := new(TickHabitResponse)
+	err := c.cc.Invoke(ctx, Habits_TickHabit_FullMethodName, in, out, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return out, nil
+}
+
+func (c *habitsClient) GetHabitStatus(ctx context.Context, in *GetHabitStatusRequest, opts ...grpc.CallOption) (*GetHabitStatusResponse, error) {
+	out := new(GetHabitStatusResponse)
+	err := c.cc.Invoke(ctx, Habits_GetHabitStatus_FullMethodName, in, out, opts...)
+	if err != nil {
+		return nil, err
+	}
+	return out, nil
+}
+
 // HabitsServer is the server API for Habits service.
-// All implementations should embed UnimplementedHabitsServer
+// All implementations must embed UnimplementedHabitsServer
 // for forward compatibility
 type HabitsServer interface {
 	// CreateHabit is the endpoint that registers a habit.
 	CreateHabit(context.Context, *CreateHabitRequest) (*CreateHabitResponse, error)
 	// ListHabits is the endpoint that returns all habits.
 	ListHabits(context.Context, *ListHabitsRequest) (*ListHabitsResponse, error)
+	// TickHabit is the endpoint to tick a habit.
+	TickHabit(context.Context, *TickHabitRequest) (*TickHabitResponse, error)
+	// GetHabitStatus is the endpoint to retrieve the status of ticks of a habit.
+	GetHabitStatus(context.Context, *GetHabitStatusRequest) (*GetHabitStatusResponse, error)
+	mustEmbedUnimplementedHabitsServer()
```

Comment:
@doniacld This is the line added by removing `require_unimplemented_...`.

In order to now implement the `HabitsServer` interface, you have to have a struct that implements that method.
The easiest way is to [embed ](https://gobyexample.com/struct-embedding) your server with the `UnimplementedHabitsServer` (as the name of the unexposed method suggests)
```go
type myServer {
  // embedding the default unimplemented server
  api.UnimplementedHabitsServer

  // other stuff
  db database...
}
```